app/templates/teachers/resource-hub-view.jade

```diff
@@ -88,13 +88,17 @@ block content
           -     faqURL += "-he";
           - if (me.get('preferredLanguage') == 'pt-BR' || me.get('preferredLanguage') == 'pt-PT')
           -     faqURL += "-pt-BR";
+          - if (me.get('preferredLanguage') == 'zh-HANS')
```

Comment:
We should show this to people using traditional Chinese, too (better than English).
```suggestion
          - if (me.get('preferredLanguage') == 'zh-HANS' || me.get('preferredLanguage') == 'zh-HANT')
```
Same with other two instances below.